README.md

```diff
@@ -1001,11 +1001,12 @@ services.AddGraphQL(b => b
     .AddSystemTextJson());
 ```
 
-Please see the 'Upload' sample for a demonstration of this technique.  Note that
-using the `FormFileGraphType` scalar requires that the uploaded files be sent only
-via the `multipart/form-data` content type as attached files.  If you wish to also
-allow clients to send files as base-64 encoded strings, you can write a custom scalar
-better suited to your needs.
+Please see the 'Upload' sample for a demonstration of this technique, which also
+demonstrates the use of the `MediaTypeAttribute` to restrict the allowable media
+types that will be accepted.  Note that using the `FormFileGraphType` scalar requires
+that the uploaded files be sent only via the `multipart/form-data` content type as
+attached files.  If you wish to also allow clients to send files as base-64 encoded
```

Comment:
```suggestion
attached files. If you also wish to allow clients to send files as base-64 encoded
```

Reply:
This repository has "properly" two spaces after every period consistently throughout all documentation.

Reply:
```suggestion
attached files.  If you also wish to allow clients to send files as base-64 encoded
```

Reply:
Merged change to move 'also'